setup.py

```diff
@@ -51,6 +51,7 @@ def get_version():
         'setuptools',
         'numpy',
         'iminuit',
+        'matplotlib'
```

Comment:
IMO matplotlib should be an optional dependency and more clearly declared as such.
My suggestion would be to put it in an `all` target like I did for iminuit:
https://github.com/iminuit/iminuit/blob/master/setup.py#L84